### PR TITLE
update file structure in readme

### PR DESCRIPTION
**Pull request by 7596ff**
while browsing, i noticed the structure was off a little bit. took care of updating for ya. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,118 +1,112 @@
...
+├── data
+│   └── js
+│       ├── payload_details.js
+│       ├── tweet_index.js
+│       ├── tweets
+│       │   ├── 2007_03.js
+│       │   ├── 2007_04.js
+│       │   ├── 2007_05.js
+│       │   ├─ ... // you get the idea, I've been on Twitter a while
+│       │   ├── 2013_0.js
```
Comment (DeMarko) [inline README.md]: what month is this?

File: README.md [hunk 2]
```diff
@@ -1,118 +1,112 @@
...
-    The file structure looks somewhat like this (as of 19.12.12):
-
-
-```
-tweets
-├── README.txt
-├── css
-│   └─ ... // provided by Twitter
-├── data
-│   ├── csv
```
Comment (DeMarko) [inline README.md:90]: Do CSVs not come in the archive dump anymore? It's been a while since I got a fresh one.

File: README.md (hunk 1 above)
Comment (7596ff) [inline README.md]: oops, seem to have forgotten a number.

File: README.md (hunk 2 above)
Comment (7596ff) [inline README.md:90]: Nop, I guess it's easier to export that way.

Comment (DeMarko) [inline README.md:90]: huh, cool

**Pull request merged**